.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+- '6'
+script: npm test
+env:
+  global:
+    secure: d7Tpzk8eq+LU/amOUlg8E2QtFalDjDmBeFUzUwJi7qeqKt0EaIKmvJdzYaIJ2B9aYfONuV/mxBLkWVVuQkycI/w88cSRh+7dGOrKKd9xL6iJkVSROt1KAGhlcrXoeJrEwYrSUfgRCz+7tfym321lauXw+4NVn7GKSgvZ4kHyiF9wUQpu/MFZVNzrzLmkdcKAfQVC9du3d/Q5g99rBwhBPme2YsdnhKfiGYFiu22e4K4tnj7ckBcCLJJp+d+d6yGmi+BhVGQalQEDz6UYmOwq+S8OzdjTHeMm0EInYJYwJ0iKc9F1hTRJYExBel+sRiLzjs2G1etIQS06LB3BTUWED0R7bmWBvBTZmdqdMAhCXpgaluH4zto3Rwe44N3m65y4nruDo9QPNUpJCseTNfaauY3qtHQH8/weJjwiIWlOM7ugdm8+ika/uZBWC/qXpv/saUJGQteSLOMeC39KyPSqZ+YVC0AjiGHR5KlxoE0Chx7DU5WEhFyWUp4Ltv2+qUpigq2nlo0ErlM4ri3qaqsWR225QvJ1bBLtq0HErh6Yj2puHRIY9Q0MjM9Tw36Alam1tGJhNTFrloYeupuU0Tx9xaR8lL5JrKJtfp+5OZyYctocaD3j/5C3prIWLHufunLSIih5aNzWIL5FgGizqhtutpA/zCXWIyaew4IkBZWHV2w=
```

Comment:
What's this do?

Reply:
These are encrypted environment variables:
https://docs.travis-ci.com/user/encryption-keys/